### PR TITLE
Update kubelet-config-file.md

### PR DESCRIPTION
**Pull request by rdxmb**
add note and link to kubeadm/kubelet-integration

fix https://github.com/kubernetes/website/issues/27191
fix https://github.com/kubernetes/website/issues/27188

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ For a trick to generate a configuration file from a live node, see
 
 ## Start a Kubelet process configured via the config file
 
+{{< note >}}
+If you use kubeadm to initialize your cluster, use the kubelet-config while creating your cluster with `kubeadmin init`. See https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/ for details.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:57]:
```suggestion
If you use kubeadm to initialize your cluster, use the kubelet-config while creating your cluster with `kubeadmin init`.
See [configuring kubelet using kubeadm](/docs/setup/production-environment/tools/kubeadm/kubelet-integration/) for details.
```

Comment (rdxmb) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:57]: ok for me. Is there a way to squash this, so I do not have two commits for this change?

Comment (rdxmb) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:57]: /label tide/merge-method-squash

**Review by neolit123 — COMMENTED** [review]
/lgtm
/sig node cluster-lifecycle

**Pull request merged**